### ISSUE TITLE
Change "REDACTED" to better text for notifications with content redacted

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#3261]
We have the following feature:

> If you’d like an extra layer of protection, there’s a `PUSH_NOTIFICATION_REDACT_CONTENT` setting available to disable any message content being sent via the push notification bouncer (i.e. message content will be replaced with `***REDACTED***`).

https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#security-and-privacy

The text "***REDACTED***" isn't really the right text for this, though. It only really makes sense from the perspective of the code that sends it. Instead, we should use text that makes sense in the actual notification the user sees.

See [chat discussion](https://chat.zulip.org/#narrow/stream/48-mobile/topic/redacted.20notifications/near/678213). Suggestions include:
* "You have a new message"
* "New message"

The main work required to resolve this is to try different texts in actual notifications on a device, and see how they look. Further thoughts:
* The code involved here is on the server, in `zerver/lib/push_notifications.py`.
* For testing server-side changes to push notifications, see [our guide](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/push-notifications.md).
* A PR for this should include screenshots of the resulting notifications. Definitely include a screenshot for iOS, and preferably also Android. (On Android, our client-side code that presents the notifications will change soon to make the notifications look quite different.)

**Comment by lukasIO** [comment]
maybe related: https://github.com/zulip/zulip/pull/8076
this is still open - not sure why, but @timabbott rebased and mentioned he knows now how this works.
I assumed the encrypted notification would make the redacted message content basically obsolete...

**Comment by timabbott** [comment]
>  It only really makes sense from the perspective of the code that sends it. 

I think actually the user who wanted this feature came up with that text as what he wanted, but I agree it doesn't make sense.

As a sidenote, this feature or something like it may be valuable even if we implement encrypting push notifications, because some organizations we've talked to have policies against having any content available on a locked phone screen.

**Comment by lukasIO** [comment]
Not sure how this works on Android, but for iOS there is an app-specific option to enable or disable the message preview on the lockscreen. 
If you disable the message preview you still get a notification to the lockscreen, but without content. If this also exists on Android I guess that would then be the desired behaviour and the "official" way of avoiding content on the lockscreen.

**Comment by gnprice** [comment]
> Not sure how this works on Android, but for iOS there is an app-specific option to enable or disable the message preview on the lockscreen.
> If you disable the message preview you still get a notification to the lockscreen, but without content. If this also exists on Android I guess that would then be the desired behaviour and the "official" way of avoiding content on the lockscreen.

Indeed, there certainly is also such a feature on Android. So I think the use of this "REDACTED" feature is primarily to hide it from the bouncer (and Google and Apple), which would be superseded by encryption.

Potentially some organizations would still want it even then, to ensure the info doesn't make it to a lock screen. I'd think an organization that wants that kind of control would, if they're on top of things, also insist on its employees using phones which it remotely manages; allow only such phones to reach services like their Zulip server; and enforce the "hide notification content on lock screen" setting globally... but there will be exceptions for a variety of reasons. So as long as the feature is there and cheap to maintain, I expect we'll keep it.
